Sources/PackageLoading/README.md

```diff
@@ -4,7 +4,7 @@ This library defines the logic which translates between the Swift package
 manager conventions and the underlying project model.
 
 The intent is that it is largely a transformation taking the input project model
-objects descripted in a manifest, applying the conventions and
+objects descripted in a manifest, applying the conventions.
```

Comment:
it: do we need the `,` comma?

Reply:
Sorry, I'm not sure I follow, where exactly should the comma be added?

Reply:
I mean, can we remove the comma?

`objects descripted in a manifest applying the conventions.`

Reply:
That's fair, I somehow assumed that the rest should be kept verbatim 😄

Reply:
thank you!